### PR TITLE
Ban command

### PR DESCRIPTION
**Pull request by WillFJS**

### CODE REVIEW

**Comment by WillFJS** [comment]
The ban and kick command needs to be integrated with the database so we can have a "modlogs" command

**Comment by MidSpike** [comment]
@WillFJS please finish each of the moderation commands separately on separate pull requests

**Comment by WillFJS** [comment]
This pull request is ready for review!

File: src/bot/commands/ban.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        } catch {
+            // ignore any errors
+        }
+
+        /* message the member in the server */
+        await message.channel.send(moderation_message_contents).catch(console.warn);
+
+        /* perform the moderation action on the member */
+        try {
+            await member.ban(reason);
```
Comment (MidSpike) [inline src/bot/commands/ban.js:63]: member.ban accepts an object as as arg
https://discord.js.org/#/docs/main/stable/class/GuildMember?scrollTo=ban

**Review by MidSpike — CHANGES_REQUESTED** [review]

**Comment by WillFJS** [comment]
Moved to #78

**Pull request closed without merging**